packages/apps/api-authorizer/src/authorization.ts

```diff
@@ -26,11 +26,19 @@ export const hasScopes = (...requiredScopes: string[]): AccessControlRule => (
   jwt,
   parameters
 ) => {
+  if (typeof jwt.payload === 'string') {
```

Comment:
Why might jwt.payload be a string at this point, and why would we not process it?

Also same question in AccessControlRule, below.

Reply:
This change came from my blatting of `yarn.lock` which led to a semver-compatible upgrade of `@types/jsonwebtoken` where opaque/non-JSON payloads were (correctly) [represented](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58158) - I'll add a comment to this effect as it's not especially clear